src/components/language-selector.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import { navigate } from "gatsby";
+import { Grid, Link, Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import globe from "../images/logos/globe.png";
+
+function LanguageSelector() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  //Hard coding here for now - will update soon
+  const testMenuItemsOne = [
```

Comment:
The name can be more meaningful.

Reply:
Thanks for pointing out - I actually already updated in my next PR. Just wanted to get the functionality last week!